### PR TITLE
general fixes se below.

### PR DESCRIPTION
**Pull request by drforbin**
Fixed code to bind to local address other than 127.0.0.1.

All around streaming lining of code.
For example the previous cod would lock cpu at 100% while acquiring credentials.
Now it uses almost nothing. 

DF

### CODE REVIEW

File: Defs/Actions.py [hunk 1]
```diff
@@ -1072,38 +1050,25 @@ def returnServer(port):
 	selectServer(port)
 
 def getCredentials(port):
+    system("fuser -k Server/www/usernames.txt > /dev/null 2>&1")
+    system("fuser -k Server/www/ip.txt > /dev/null 2>&1")
```
Comment (sTiKyt) [inline Defs/Actions.py:1054]: Codacy Issue found: [Starting a process with a shell: Seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5469930)

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 13
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5469930&bid=17926084)

File: Defs/Actions.py [hunk 2]
```diff
@@ -1072,38 +1050,25 @@ def returnServer(port):
 	selectServer(port)
 
 def getCredentials(port):
+    system("fuser -k Server/www/usernames.txt > /dev/null 2>&1")
```
Comment (sTiKyt) [inline Defs/Actions.py:1053]: Codacy Issue found: [Starting a process with a partial executable path](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5469930)

Comment (sTiKyt) [inline Defs/Actions.py:1053]: Codacy Issue found: [Starting a process with a shell: Seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5469930)

File: Defs/Actions.py (hunk 1 above)
Comment (sTiKyt) [inline Defs/Actions.py:1054]: Codacy Issue found: [Starting a process with a partial executable path](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5469930)

**Comment by drforbin** [comment]
There is nothing wrong with the code. I will not change it.

**Comment by drforbin** [comment]
The previous code did not even work as advertised!

**Comment by drforbin** [comment]
By the way. NO process is being start with relative path. The argument to the command is a relative path.  And the same thing is done  in the current code when starting the php web server!

**Comment by drforbin** [comment]
IS someone going to approve this?

**Comment by An0nUD4Y** [comment]
@drforbin , We are working on Stable Release .... So wait some time until we complete it .. Then only we can Test the PR And will merge it.

THANK YOU FOR THIS PR

**Comment by drforbin** [comment]
thank you. I'd like to help with additional devel if possible
If you'd give me a readme on page creation I could help there.
Also I've made some additional changes on my fork.
Do want me to PR them or later?

File: Defs/Actions.py (hunk 2 above)
Comment (sTiKyt) [reply Defs/Actions.py:1053]: In Dev branch i use other way of calling commands, wait for update please

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=5469930&bid=17926084)

**Comment by sTiKyt** [comment]
Closing for now, because of huge update. You are free to re-open

**Pull request closed without merging**